src/deps.ts

```diff
@@ -38,3 +37,4 @@ export {
   resolve as resolveImportMap,
 } from "https://esm.sh/@import-maps/resolve@1.0.1";
 export type { ParsedImportMap } from "https://esm.sh/@import-maps/resolve@1.0.1";
+export { createGraph } from "https://deno.land/x/deno_graph@0.25.0/mod.ts";
```

Comment:
Do we want to be loading the graph for runtime scenarios? I thought we put it in deps.dev for that reason?

Reply:
It's used for module preloading

Reply:
Ah I see. We could probably revisit this later, as we could dump the graph to json during, and use that.

Reply:
Yea, I noticed because this dep has a `wasm` import, that it slows the start of our server a bit. I think we should only include this dep if the preload headers are enabled.